zwave_js_server/model/value.py

```diff
@@ -258,7 +264,7 @@ class ConfigurationValue(Value):
     def configuration_value_type(self) -> ConfigurationValueType:
         """Return configuration value type."""
         if self.metadata.type == "number":
-            if self.metadata.states:
+            if self.metadata.states and not self.metadata.allow_manual_entry:
```

Comment:
This would add support for the scenario you described on Discord, but it might be better to have a new `ConfigurationValueType` for this type of property and then update the `async_set_config_parameter` function to support it (changes required would be minimal).

We should also add tests

Reply:
Tests added.

Reply:
New `ConfigurationValueType` added.